pom.xml

```
@@ -45,6 +46,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <flow.version>24.1-SNAPSHOT</flow.version>
+        <hilla.version>2.0.0</hilla.version>
```

We should target Hilla 2.1 here to be sure the kit works with the latest platform.
```suggestion
        <hilla.version>2.1-SNAPSHOT</hilla.version>
```